### DISCUSSION TITLE
Template for baseline_table.dat not found

### DISCUSSION THREAD

**Discussion by Fsy0325** [insarlab/MintPy#1262] [Q&A]
After the GMTSAR calculation generates the interference pattern, I want to use mintpy to perform timing analysis. Is there a template for the baseline_table.dat file in the example directory? How is it generated?
![image](https://github.com/user-attachments/assets/c683cd9d-0bc8-429e-9dfa-83d6580fb121)

**Comment by welcome** [comment]
👋  Thanks for opening your first issue here! Please filled out the template with as much details as possible. We appreciate that you took the time to contribute!
Make sure you read our [contributing guidelines](https://github.com/insarlab/MintPy/blob/master/docs/CONTRIBUTING.md).

**Comment by yunjunz** [comment]
You could find an example here: https://mintpy.readthedocs.io/en/latest/demo_dataset/#sentinel-1_on_san_francisco_bay_with_gmtsar. @Xiaohua-Eric-Xu should know the details better.
